README.md

```diff
@@ -523,19 +523,19 @@ gis:load-dataset *file*
 ```
 
 
-Loads the given data file, re-projecting the data as necessary if a
-global projection is defined and if the data file itself has an
-associated .prj file, then reports the resulting dataset.
+Loads the given data file, re-projecting the data as necessary. 
 
-If no ".prj" file is present, then `load-dataset`
-assumes that the projection of the data being loaded is the same as
+For ESRI shapefiles and ESRI grid files, if there is a ".prj" file 
+associated with the file, then `load-datset` will consult that file
+and re-project to the current global projection if needed. If no ".prj"
+file is found, then the data is assumed to use the same projection as
 the current global coordinate system.
 
-Relative paths are resolved relative to the location of the current
-model, or the user's home directory if the current model
-hasn't been saved yet.
```

Comment:
I see that this relative path info is also listed in the `set-coordinate-system` docs.  Is there a reason you don't think it should be included here as well?  It's still valid even with the GeoJSON changes, correct?